examples/gnrc_border_router/Makefile

```diff
@@ -41,6 +41,9 @@ USEMODULE += ps
 # to auto-configure that upstream DNS server.
 #USEMODULE += sock_dns
 
+# The number of native (emulated) ZigBee/6LoWPAN devices
+ZEP_DEVICES ?= 1
```

Comment:
I would prefer to move this into the else branch of the "not `native`"-check before [this line](https://github.com/RIOT-OS/RIOT/pull/13810/files#diff-8790a8833e3d374cd14889b9f0e69a2cL90)